src/cmd/sql_cmd.h

```diff
@@ -74,6 +74,9 @@ const std::string VERSION = std::to_string(OPENMLDB_VERSION_MAJOR) + "." +  // N
 std::string db = "";  // NOLINT
 ::openmldb::sdk::DBSDK* cs = nullptr;
 ::openmldb::sdk::SQLClusterRouter* sr = nullptr;
+using VariableMap = std::map<std::string, std::string>;
+VariableMap session_variables = {VariableMap::value_type("execute_mode", "online"),
+                                 VariableMap::value_type("performance_sensitive", "true")};
```

Comment:
Actually, we will eliminate performance-sensitive mode in #877 .

Reply:
We have to resolve conflicts after #796 has been merged.

Reply:
OK, I'll fix it.